homeassistant/components/zha/__init__.py

```diff
@@ -105,6 +108,12 @@
 
     Will automatically load components to support devices found on the network.
     """
+    if config_entry.data.get(ENABLE_QUIRKS):
+        # needs to be done here so that the ZHA module is finished loading
+        # before zhaquirks is imported
+        # pylint: disable=W0611, W0612
+        import zhaquirks  # noqa
```

Comment:
Does this have any effect? We're not using the import anywhere, it seems.

Reply:
Yes, Zigpy automatically picks up all classes that extend one of its base classes and it adds them to a registry.